tests/Unit/ScoutSearchCommandBuilderTest.php

```diff
@@ -76,7 +76,7 @@ public function test_it_sets_data_based_on_the_scout_builder(string $method, mix
         $builder = Mockery::mock(Builder::class);
         $builder->index = self::TEST_INDEX;
 
-        $setter = mb_strtolower($method);
+        $setter = lcfirst($method);
```

Comment:
Is this because the i in whereIn still needs to be capitalized?

Reply:
that's exactly it. Methods aren't case sensitive in php so that isn't affected, but the `$setter` sets the wrong property on the mock builder.